### ISSUE TITLE
Where can I find the custom loss functions

### ISSUE THREAD

**Issue by ezeid** [broadinstitute/ml4h#607]
I am trying to load the lvh model: ecg_rest_raw_age_sex_bmi_lvm_asymmetric_loss.h5 with tensorflow 
model = load_model(model_path), but it complains about a bunch of custom loss functions.

'symmetric_logcosh_loss_stable': symmetric_logcosh_loss_stable,
 'loss_fxn': loss_fxn,
'asymmetric_outlier_mse':asymmetric_outlier_mse,
 'lvh_from_indexed_lvm_weighted_loss': lvh_from_indexed_lvm_weighted_loss,
  'no_lvh_precision': no_lvh_precision,
 'left_ventricular_hypertrophy_precision': left_ventricular_hypertrophy_precision,
'no_lvh_recall': no_lvh_recall
...

I could not find these functions defined in the repo. Where can I find it?
